controls/bsi_sys_1_6.yml

```diff
@@ -0,0 +1,412 @@
+# In BSI Basic Protection are multiple Requirements in one control.
+# i.e. there are multiple sentences, some including a RFC2119 keyword
+# Since we must increase granularity to create a precise control,
+# we number each sentence with a RFC2119 keyword as a section, grouping sentences, which are logically connected.
+# we number inline in brackets, so the lookup is easy
+# we reference these numbers in comments over each rule or group of rules
```

Comment:
Yes, since we only setup the control file for sys.1.6 and are currently prioritizing app.4.4 there is nothing like that currently in this pr.

nevertheless there is an example in another PR right here: https://github.com/ComplianceAsCode/content/blob/2439d34019425c375c0bb7689649f61df9ef6845/controls/bsi_app_4_4.yml#L70-L100

We might overload the control file with our approach so feel free to comment :)

Reply:
Thank you for the example.

> We might overload the control file with our approach so feel free to comment :)

This seems fine for now. But I see how this could lengthen the control file.

Regardless, this reminds me that control files can be split into multiple files in a directory, see for example the cis_ocp4 [file](https://github.com/ComplianceAsCode/content/blob/master/controls/cis_ocp_1_4_0.yml) and [directory](https://github.com/ComplianceAsCode/content/tree/master/controls/cis_ocp_1_4_0).